src/test/java/com/microsoftopentechnologies/windowsazurestorage/helper/CredentialMigrationTest.java

```diff
@@ -44,62 +44,71 @@ public class CredentialMigrationTest {
     "<storageAccName>12345</storageAccName>\n" +
     "<storageAccountKey>abcdef</storageAccountKey>\n" +
     "<blobEndPointURL>http://test2/</blobEndPointURL>\n" +
+    "<cdnEndPointURL>http://cdn-test2/</cdnEndPointURL>\n" +
     "</com.microsoftopentechnologies.windowsazurestorage.beans.StorageAccountInfo>\n" +
     "</storageAccounts>\n" +
     "</com.microsoftopentechnologies.windowsazurestorage.AzureStoragePublisher_-WAStorageDescriptor>";
-    
+
     private static final String wrongConfigContent = "<?xml version='1.0' encoding='UTF-8'?>\n" +
     "<com.microsoftopentechnologies.windowsazurestorage.AzureStoragePublisher_-WAStorageDescriptor plugin='windows-azure-storage@0.3.3-SNAPSHOT'>\n" +
     "<storageAccounts>\n" +
     "<storageAccName>abcdef</storageAccName>\n" +
     "<storageAccountKey>12345</storageAccountKey>\n" +
     "<blobEndPointURL>https://blob.core.windows.net/</blobEndPointURL>\n" +
+    "<blobEndPointURL>https://cdn-resource-name.azureedge.net/</blobEndPointURL>\n" +
     "</storageAccounts>\n" +
     "</com.microsoftopentechnologies.windowsazurestorage.AzureStoragePublisher_-WAStorageDescriptor>";
-    
+
     @Rule
     public TemporaryFolder testFolder = new TemporaryFolder();
-    @ClassRule 
+    @ClassRule
     public static JenkinsRule j = new JenkinsRule();
 
     @Before
     public void setUp() {
         jenkinsInstance = Jenkins.getInstance();
-        
     }
+
     /**
      * Test of upgradeStorageConfig method, of class CredentialMigration.
      */
     @Test
     public void testUpgradeStorageConfig() throws Exception {
         System.out.println("upgradeStorageConfig");
-        
+
         String storageAccount = "abcdef";
         String storageAccountKey = "12345";
         String storageBlobURL = "http://test1/";
+        String storageCdnURL = "http://cdn-test1/";
         File configFile = testFolder.newFile(Constants.LEGACY_STORAGE_CONFIG_FILE);
         FileUtils.writeStringToFile(configFile, correctConfigContent);
         FileUtils.copyFileToDirectory(configFile, jenkinsInstance.root);
 
         CredentialMigration.upgradeStorageConfig();
-        
+
         CredentialsStore s = CredentialsProvider.lookupStores(jenkinsInstance).iterator().next();
 
         assertEquals(2, s.getCredentials(Domain.global()).size());
-                
-        AzureStorageAccount.StorageAccountCredential u = new AzureStorageAccount.StorageAccountCredential(storageAccount, storageAccountKey, storageBlobURL);
+
+        AzureStorageAccount.StorageAccountCredential u = new AzureStorageAccount.StorageAccountCredential(storageAccount, storageAccountKey, storageBlobURL, storageCdnURL);
         AzureStorageAccount storageCred = CredentialsMatchers.firstOrNull(CredentialsProvider.lookupCredentials(AzureStorageAccount.class, jenkinsInstance, ACL.SYSTEM, Collections.<DomainRequirement>emptyList()),
                 CredentialsMatchers.withId(u.getId()));
 
+        System.out.println("BELOW THIS LINE");
```

Comment:
revert the changes to this file if they aren't needed, i wouldn't expect to have to change this

Reply:
Good catch - addressed in latest commits.